### ISSUE TITLE
Autocomplete of anonymous `function()` eats following closing brace

### ISSUE THREAD

**Issue by rulatir** [EmmyLua/IntelliJ-EmmyLua#366]
Environment（环境）
--------------
   name   |   version  
----------|----------
 IDEA version         | PhpStorm 2020.1.3
 EmmyLua version | 1.3.3.150-IDEA201
 OS                        | Linux 64bit

What are the steps to reproduce this issue?（重现步骤？）
------------------------------------------------
1. Have this (`|` - caret):
```
describe("This thing", |)
```
2. Type `function()`

What happens?（出现什么问题？）
-------------
Autocomplete result:
```
describe("This thing", function(|) end
```
(note missing closing brace)

What were you expecting to happen?（期望？）
-----------------------------------------
Autocomplete result:
```
describe("This thing", function(|) end)
```

Any other comments?（其它说明）
------------------------
This requires manual fixing every time, and it takes about the same number of keystrokes as would be necessary to type ` end` by hand and move the caret between the braces.

**Comment by rulatir** [comment]
This really wrecks concentration when writing busted unit tests. Is there anything we can do to attract the author's attention to this issue?

**Comment by tangzx** [comment]
fixed, see #337
